tests/test_input.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+from kvstore.input import parse_input
+
+ERROR = "error"
+
+def test_parse_input():
+    test_cases = [
+        # nonsense commands
+        {"input": "asdfasdf asdf", "expected": ERROR},
+        {"input": "get hello=123", "expected": ERROR},
+        {"input": "get ===", "expected": ERROR},
+        {"input": "get    abc", "expected": ERROR},
```

Comment:
As a user, whitespace being significant would surprise me, but I don't know how hard that is to fix.